.storybook/webpack.config.js

```diff
@@ -21,12 +21,34 @@ const babelLoader = {
   },
 };
 
+// const docgenLoader = {
+//   loader: require.resolve('react-docgen-typescript-loader'),
+//   options: {
+//     tsconfigPath: path.join(__dirname, 'tsconfig.json'),
+//     skipPropsWithoutDoc: true,
+//   },
+// };
```

Comment:
It looks like props coming from certain paths can be excluded:
```ts
const docgenLoader = {
  loader: require.resolve('react-docgen-typescript-loader'),
  options: {
    propFilter: prop => {
      // Filter out props from styled-components
      if (prop.name === 'as' || prop.name === 'ref' || prop.name === 'theme') {
        return false;
      }

      if (prop.parent == null) {
        return true;
      }

      // Filter out props which type definition is placed in react package
      return prop.parent.fileName.indexOf('node_modules/@types/react') < 0;
    }
  }
};
```

Reply:
Yeah I saw this. We can use this but we will not be able to use `HappyPack` for the loader since it will JSON sanitize the options. I can look at running it separate from `HappyPack`